modules/get-started/pages/whats-new.adoc

```diff
@@ -15,6 +15,10 @@ Redpanda now provides a FIPS 140-compliant binary to support running Redpanda in
 
 Redpanda's xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry] now supports JSON serialization format, in addition to Avro and Protobuf. You can use a https://json-schema.org/[JSON Schema^] format to validate messages and ensure compatibility between schema versions.
 
+== Minimum TLS version
```

Comment:
Maybe we should group this with certificate revocation lists under a parent heading of TLS enhancement?

```
== TLS enhancement

* *Minimum TLS version*: For enhanced security, Redpanda now has a cluster configuration property to xref:manage:security/encryption.adoc#manage-the-minimum-tls-version[control the minimum TLS version] to use for encryption. By default, xref:reference:properties/cluster-properties.adoc#tls_min_version[`tls_min_version`] is set to `v1.2`, which prevents client applications from using older TLS versions to connect to your clusters.
* *Certificate revocation lists*: When enabling TLS encryption for the Kafka, Admin, HTTP Proxy or Schema Registry APIs, you can now xref:manage:security/encryption.adoc[add a certificate revocation list (CRL)] to your configuration. Redpanda uses the CRL to check and reject connections from entities using certificates already revoked by a certificate authority.
```